src/main.rs

```diff
@@ -1,26 +1,39 @@
-use std::io::{self, stdout, BufRead, Write};
+use std::{
+    env,
+    io::{self, stdout, BufRead, Write},
+};
 use yaiwr::Calc;
 
 fn main() {
-    let stdin = io::stdin();
-    loop {
-        print!("👉 ");
-        stdout().flush().ok();
-        match stdin.lock().lines().next() {
-            Some(Ok(ref l)) => {
-                if l.trim().is_empty() {
-                    continue;
-                }
-                let ast = Calc::from_str(l);
-                match ast {
-                    Ok(opcode) => match Calc::eval(opcode) {
-                        Ok(i) => println!("Result: {}", i),
-                        Err(msg) => eprintln!("Evaluation error: {}", msg),
-                    },
-                    Err(msg) => eprintln!("Evaluation error: {}", msg),
+    let args: Vec<String> = env::args().collect();
+
+    if args.len() > 1 {
+        eval_print(&args[1]);
+    } else {
+        let stdin = io::stdin();
+        loop {
+            print!("👉 ");
+            stdout().flush().ok();
+            match stdin.lock().lines().next() {
```

Comment:
Can expressions span multiple lines in your language?

E.g. is this a valid input?:
```
2+
1
```

Reply:
It's not super important actually. Never mind.

Reply:
Yeah, I can add support for it in the future, my intention with this PR was to allow me to test its basic functionality faster so I intentionally kept it very simple